components/playgroundChatbot.tsx

```diff
@@ -62,205 +51,20 @@ export default function PlaygroundChatbot(props: {
       );
     })();
   }, []);
-  useEffect(() => {
-    if (!profile) return;
-    (async () => {
-      // Get the usage count for the user
-      if (
-        profile?.organizations!.is_paid.length === 0 ||
-        !profile?.organizations!.is_paid[0].is_premium
-      ) {
-        // Below is the number of 1st messages sent by the organization's users
-        const usageRes = await supabase
-          .from("chat_messages")
-          .select("*", { count: "exact" })
-          .eq("org_id", profile?.organizations!.id)
-          .eq("conversation_index", 0)
-          .eq("role", "user");
-        if (usageRes.error) throw new Error(JSON.stringify(usageRes));
-        let numQueriesMade = usageRes.count ?? 0;
-        const messagesSent = usageRes.data.map((message) => message.content);
-        // This accounts for the suggestions of a preset. The preset adds 3 messages to the DB
-        if (
-          numQueriesMade &&
-          (suggestions1.every((s) => messagesSent.includes(s)) ||
-            suggestions2.every((s) => messagesSent.includes(s)))
-        ) {
-          // 3 suggestions, so reduce by 3
-          numQueriesMade -= 3;
-        }
-        setUsageLevel(numQueriesMade);
-      }
-    })();
-  }, [profile]);
 
-  const [conversationId, setConversationId] = useState<number | null>(null);
-  const [devChatContents, setDevChatContents] = useState<StreamingStepInput[]>(
-    []
-  );
-  const [userText, setUserText] = useState<string>("");
-  const [loading, setLoading] = useState<boolean>(false);
   const [devMode, setDevMode] = useState<boolean>(false);
-  const killSwitchClicked = useRef(false);
-  const submitButtonClickable = !props.submitErrorMessage && userText;
-
-  const onChatSubmit = useCallback(
-    async (chat: StreamingStepInput[]) => {
-      setDevChatContents(chat);
-      setUsageLevel((prev) => prev + 1);
-      if (loading || alreadyRunning.current) return;
-      alreadyRunning.current = true;
-      setLoading(true);
-      const response = await fetch("/api/v1/answers", {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json",
-          Authorization: `Bearer ${profile!.organizations!.api_key}`,
-          sessionToken: session?.access_token ?? "",
```

Comment:
Removal of this means you need to use the `cookie` in `/answers` to get the session token now :smile: